packages/core/useMouse/index.ts

```diff
@@ -31,6 +31,14 @@ export interface UseMouseOptions extends ConfigurableWindow, ConfigurableEventFi
    */
   touch?: boolean
 
+  /**
+   * Customize scrollTarget to listen to `scroll` events for type `page`
+   * Set to `null` or `undefined` to disable
+   *
+   * @default null
```

Comment:
This does not align with the actual implementation. We could say this is enabled by default when type is `page`, and is only effective on type `page`?

Reply:
I'm not pretty sure that adding a scrollTarget is appropriate. Since `pageX` and `pageY` may not change if the `scrollTarget` is not `window`, it may be better to replace it with `scroll?: boolean`, which defaults to true when `type = "page"`.

When the scroll target is not a `window` but a custom element (e.g. `#app`) , user may need to customize `extractors` in order to get the mouse distance from the top coordinate.

In this case, there's a limit to what vueuse can handle, and it may only be able to cover the case where the `scrollTarget` is `window` and the `type` is `page`. 

I'm more confused about the API design here, so I wonder if I can change the options to `scroll?: boolean` and only handle the case where the `window` is the `scrollTarget`.

Reply:
Yes, I think `scroll?: boolean` makes sense

Reply:
When the user really needs to use reactive mouse position in a custom element with the help of `useMouse`, it won't be possible. If we use `scrollTarget` with a user-defined `type` extractor, we can achieve it, but it's a bit tricky for the user to implement.

In this situation, use `useElementBounding` and `useMouse` together can also resolve the problem(like the solution in this [PR](https://github.com/CatsJuice/AFFiNE.pro/commit/a5d56d799e1494ba03431a99ab13f9245fc6c95d))

I'll change the option to `scroll?: boolean` first in current PR!

:)